README.md

```diff
@@ -1,36 +1,138 @@
+<h4 align='center'><i>this repo is under development and not stable!</i></h4>
 
-# napari
-[![License](https://img.shields.io/pypi/l/napari.svg)](https://github.com/napari/napari/raw/master/LICENSE)
-[![Build Status](https://api.cirrus-ci.com/github/Napari/napari.svg)](https://cirrus-ci.com/napari/napari)
-[![Python Version](https://img.shields.io/pypi/pyversions/napari.svg)](https://python.org)
-[![PyPI](https://img.shields.io/pypi/v/napari.svg)](https://pypi.org/project/napari)
-[![PyPI - Downloads](https://img.shields.io/pypi/dm/napari.svg)](https://pypistats.org/packages/napari)
-[![Development Status](https://img.shields.io/pypi/status/napari.svg)](https://github.com/napari/napari)
+<h1 align='center'>napari</h1>
 
----
+<h4 align='center'>multi-dimensional image viewer for python</h4>
 
-napari is a fast multi-dimensional image viewer for Python.
+<p align="center">
+  <a href="https://github.com/napari/napari/raw/master/LICENSE"><img src="https://img.shields.io/pypi/l/napari.svg" alt="License"></a>
+  <a href="https://cirrus-ci.com/napari/napari"><img src="https://api.cirrus-ci.com/github/Napari/napari.svg" alt="Build Status"></a>
+  <a href="https://python.org"><img src="https://img.shields.io/pypi/pyversions/napari.svg" alt="Python Version"></a>
+  <a href="https://pypi.org/project/napari"><img src="https://img.shields.io/pypi/v/napari.svg" alt="PyPI"></a>
+  <a href="https://pypistats.org/packages/napari"><img src="https://img.shields.io/pypi/dm/napari.svg" alt="PyPI - Downloads"></a>
+  <a href="https://github.com/napari/napari"><img src="https://img.shields.io/pypi/status/napari.svg" alt="Development Status"></a>
 
-This project is in the **pre-alpha** stage; **expect breaking changes** to be made from patch to patch.
+</p>
+<br>
 
-## Installation
+**napari** is a fast interactive multi-dimensional image viewer for Python. It's designed for browsing, annotating, and analyzing large multi-dimensional images. It's built on top of `PyQt` (for the GUI), `vispy` (for performant GPU-based rendering), and the scientific Python stack (`numpy`, `scipy`).
 
-napari can be installed on most operating systems with `pip install napari`.
+We're developing **napari** in the open! But the project is in a **pre-alpha** stage. You can follow progress on this repository, test out new versions as we release them, and contribute ideas and code. Expect **breaking changes** from patch to patch.
 
-### Downloading OpenGL
+## installation
 
-Windows and Linux users may need to install the [Mesa OpenGL](https://www.mesa3d.org/) library.
+napari can be installed on most Mac OS X and Linux systems with Python 3.6 or 3.7 by calling 
 
-`vispy` provides an easily-downloadable (but out-of-date) Windows version for [64-](https://github.com/vispy/demo-data/raw/master/mesa/opengl32_mingw_64.dll) and [32-bit](https://github.com/vispy/demo-data/raw/master/mesa/opengl32_mingw_64.dll) operating systems.
+```sh
+$ pip install napari
+```
```

Comment:
maybe add a note about installing the github version?

Reply:
ah good call! is this right?

---
To install from the `master` branch on github use
```
pip install git+https://github.com/napari/napari
```
To clone the repository locally and install in editable mode use
```
git clone https://github.com/napari/napari.git
cd napari
pip install -e .
```

Reply:
yep! =)

Reply:
👍